droidlet/interpreter/robot/swarm_loco_interpreter.py

```diff
@@ -0,0 +1,50 @@
+from droidlet.interpreter.robot import LocoInterpreter
+from droidlet.interpreter.robot import swarm_tasks
+from typing import Tuple, Dict, Any, Optional
+
+class SwarmLocoInterpreter(LocoInterpreter):
```

Comment:
are we only supporting stop and resume in robots?

Reply:
For now, it's move/stop/resume. I haven't looked into other locobot swarm tasks which might require specific locobot swarm task distribution.